docs/reference/cat/shards.asciidoc

```diff
@@ -119,24 +386,4 @@ twitter 0 r STARTED    3014 31.1mb 192.168.56.30 bGG90GE
 twitter 0 r STARTED    3014 31.1mb 192.168.56.20 I8hydUG
 twitter 0 r UNASSIGNED ALLOCATION_FAILED
 ---------------------------------------------------------------------------
-// TESTRESPONSE[non_json]
-
-[float]
-[[reason-unassigned]]
-==== Reasons for unassigned shard
-
-These are the possible reasons for a shard to be in a unassigned state:
-
-[horizontal]
```

Comment:
Moved to https://github.com/elastic/elasticsearch/pull/45392/files#diff-5f7818b093901b6114a63793bc045f03R246